### PR TITLE
Crypto org Intergration

### PR DESCRIPTION
**Pull request by crypto-facs**
### Type

Crypto.org integration that includes:
- Send and receive
- Memo Field
- Account Sub Header

### CODE REVIEW

File: src/live-common-setup.js [hunk 1]
```diff
@@ -51,6 +52,7 @@ setSupportedCurrencies([
   "bitcoin_testnet",
   "ethereum_ropsten",
   "cosmos_testnet",
+  "crypto_org_croeseid",
   "elrond",
 ]);
```
Comment (gre) [inline src/live-common-setup.js:57]: Can you not enable the coin in this PR ? thanks

**Review by gre — CHANGES_REQUESTED** [review]
apart from this feedback, all is good

**Review by gre — COMMENTED** [review]
src/families/crypto_org/AccountSubHeader.js empty file?

File: src/config/urls.js [hunk 2]
```diff
@@ -136,6 +136,9 @@ export const urls = {
   elrond: {
     website: "https://elrond.com",
   },
+  elrond: {
```
Comment (hzheng-ledger) [inline src/config/urls.js:139]: Change "elrond" to "cryptoOrg" plz

**Review by hzheng-ledger — CHANGES_REQUESTED** [review]
See my comment plz

File: src/live-common-setup.js (hunk 1 above)
Comment (crypto-facs) [reply src/live-common-setup.js]: Done

**Comment by crypto-facs** [comment]
> src/families/crypto_org/AccountSubHeader.js empty file?

Sorry I forgot to commit this file! I just did it. Fixed the other comments as well so should be good for another review! Thanks 🙏

File: src/const/navigation.js [hunk 3]
```diff
@@ -189,6 +189,9 @@ export const ScreenName = {
   CosmosClaimRewardsValidationError: "CosmosClaimRewardsValidationError",
   CosmosClaimRewardsValidationSuccess: "CosmosClaimRewardsValidationSuccess",
 
+  //crypto_org
+  CryptoEditMemo: "CryptoEditMemo",
```
Comment (haammar-ledger) [inline src/const/navigation.js:193]: Please rename to `CryptoOrgEditMemo` (in other places too). Thanks.

Comment (crypto-facs) [reply src/const/navigation.js]: Done 🙏

**Review by ghost — APPROVED** [review]
:+1:

**Pull request merged**